src/09-word-problems.js

```diff
@@ -127,7 +174,38 @@ function compareLocations(address1, address2) {}
     //>   },
     //> ];
  */
-function gradeAssignments(assignments) {}
+function gradeAssignments(assignments) {
+    //
```

Comment:
Nice work on this function!

This logic is starting to get a little more complicated. Could you refactor this to split up your code into helper functions?